src/components/Panels/WavefrontSensors/Logs/Logs.tsx

```diff
@@ -13,7 +14,7 @@ export default function Logs() {
   useEffect(() => {
     if (data?.logMessage) {
       // Give each message a unique id
-      const msg = { id: self.crypto.randomUUID?.(), ...data.logMessage };
+      const msg = { id: uuid(), ...data.logMessage };
```

Comment:
Did this not work in firefox? I feel like I tested it and it worked. If it's just generating unique id's something like [nanoid](https://github.com/ai/nanoid) might also be a good fit. But this works just as well.

According to [caniuse](https://caniuse.com/mdn-api_crypto_randomuuid) we should be able to use this

Reply:
Nope, it wasn't working, actually caniuse also says it should work in the chrome version I am using (127), but for some reason I was getting always id = undefined.

Reply:
Oh it's probably because of this:

> Secure context: This feature is available only in [secure contexts](https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts) (HTTPS), in some or all [supporting browsers](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID#browser_compatibility).

👍

Reply:
Didn't know that, maybe we can go back to crypto.randomUUID after enabling the TLS certificate